docs/environment.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
This article does not belong to the benchmark repo, it contains general configuration info of an NLB environment.

Reply:
The article is heavily benchmark specific in many places, but also has generally useful information. I suggest to leave it as it is and create a new one (based on this) for general purposes in the appropriate repository.

Reply:
Ok, let's check this again.